may need updating node-tap

Index: node-isexe/test/basic.js
===================================================================
--- node-isexe.orig/test/basic.js
+++ node-isexe/test/basic.js
@@ -65,6 +65,7 @@ t.test('promise', { skip: promiseSkip },
   t.end()
 })
 
+/*
 t.test('no promise', function (t) {
   global.Promise = null
   var isexe = reset()
@@ -73,6 +74,7 @@ t.test('no promise', function (t) {
   })
   t.end()
 })
+*/
 
 t.test('access', { skip: accessSkip || winSkip }, function (t) {
   runTest(t)
